RFC/src/RFC-0171_MessageSerialisation.md

```diff
@@ -135,52 +135,54 @@ Several candidates fulfil these properties to some degree.
+Similar to [Message Pack](#message-pack), but also requires schemas to be written and compiled. Serialization 
+performance and size
+are similar to Message Pack. It Can work with ZMQ, but is better designed to be used with gRPC.
 
 #### [Cap'n Proto](http://kentonv.github.io/capnproto/)
 
 Similar to [Protobuf](#protobuf), but claims to be much faster. Rust is supported.
 
-#### Hand-rolled serialisation
+#### Hand-rolled Serialization
 
 [Hintjens recommends](http://zguide.zeromq.org/py:chapter7#Serialization-Libraries) using hand-rolled serialization for
-bulk messaging. While Pieter usually offers sage advice, I'm going to argue against using custom serialisers at this
-point in time for the following reasons:
-* We're unlikely to improve hugely over MessagePack
-* Since serde does 95% of our work for us with MessagePack, there's significant development overhead (and new bugs)
+bulk messaging. While Pieter usually offers sage advice, I'm going to argue against using custom serializers at this
+stage for the following reasons:
+
+* We're unlikely to improve hugely over MessagePack.
+* Since Serde does 95% of our work for us with MessagePack, there's a significant development overhead (and new bugs)
   involved with a hand-rolled solution.
-* We'd have to write de/serialisers for every language that wants Tari bindings; whereas every major language has a
+* We'd have to write de/serializers for every language that wants Tari bindings; whereas every major language has a
   MessagePack implementation.
 
-### Serialisation in Tari
+### Serialization in Tari
 
-Deciding between these protocols is largely a matter of preference, since there isn't that much to choose between them.
-Given that ZMQ is used in other places in the Tari network; MessagePack looks to be a good fit while offering a compact
-data structure and highly performant de/serialisation. In Rust in particular, there's first-class support for MessagePack
-in serde.
+Deciding between these protocols is largely a matter of preference, since there isn't that much difference between them.
+Given that ZMQ is used in other places in the Tari network, MessagePack looks to be a good fit while offering a compact
+data structure and highly performant de/serialization. In Rust, in particular, there's first-class support for MessagePack
+in Serde.
 
 For human-readable formats, it makes little sense to deviate from JSON. For copy-paste semantics, the extra compression
 that Base64 offers over raw hex or Base58 makes it attractive.
 
 Many Tari data types' binary representation will be the straightforward MessagePack version of each field in the related
-Struct. In these cases, as straightforward `#[derive(Deserialize, Serialize)]` is all that is required to make the data
-structure able to be sent over the wire.
+struct. In these cases, a straightforward `#[derive(Deserialize, Serialize)]` is all that is required to enable the data
```

Comment:
```suggestion
`struct`. In these cases, a straightforward `#[derive(Deserialize, Serialize)]` is all that is required to enable the data
```

Reply:
We can put this in backticks to mark it as code as a `struct` is a Rust data type, FYI see [here](https://doc.rust-lang.org/rust-by-example/custom_types/structs.html).